specs/test_formats/epoch_processing/README.md

```diff
@@ -0,0 +1,34 @@
+# Epoch processing tests
+
+The different epoch sub-transitions are tested individually with test handlers.
+The format is similar to block-processing state-transition tests.
+There is no "change" factor however, the transitions are pure functions with just the pre-state as input.
+Hence, the format is shared between each test-handler. (See test condition documentation on how to run the tests.)
+
+## Test case format
+
+```yaml
+description: string    -- description of test case, purely for debugging purposes
+bls_required: bool     -- optional, true if the test validity is strictly dependent on BLS being ON. False otherwise.
+bls_ignored: bool      -- optional, true if the test validity is strictly dependent on BLS being OFF. False otherwise.
+pre: BeaconState       -- state before running the sub-transition
+post: BeaconState      -- state after applying the epoch sub-transition.
+```
+
+Note: if both `bls_required` and `bls_ignored` are false (or simply not included),
+ then the test consumer can freely choose to run with BLS ON or OFF.
+One may choose for OFF for performance reasons during repeated testing. Otherwise it is recommended to run with BLS ON.
```

Comment:
Suggestion: the combination of `bls_required` and `bls_ignored` can be simplified to using only one required parameter `bls_setting: int` to indicate all the cases.
```
bls_setting=0: restrict to off
bls_setting=1: restrict to on
bls_setting=2: other cases, arbitrary
```

Reply:
See 6ffd416